public/index.html

```diff
@@ -59,6 +59,175 @@ <h1>
+      <div class="card">
+        <div class="project-image"></div>
+        <div class="project-info">
+          <div class="project-name">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+          </div>
+          <div class="skills">
+            <div>Ruby on rails</div>
+            <div>CSS</div>
+            <div>JavaScript</div>
+            <div>html</div>
+          </div>
+          <div class="project-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <!-- 4th Card -->
+      <div class="card">
+        <div class="project-image"></div>
+        <div class="project-info">
+          <div class="project-name">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+          </div>
+          <div class="skills">
+            <div>Ruby on rails</div>
+            <div>CSS</div>
+            <div>JavaScript</div>
+            <div>html</div>
+          </div>
+          <div class="project-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <!-- 5th Card -->
+      <div class="card">
+        <div class="project-image"></div>
+        <div class="project-info">
+          <div class="project-name">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+          </div>
+          <div class="skills">
+            <div>Ruby on rails</div>
+            <div>CSS</div>
+            <div>JavaScript</div>
+            <div>html</div>
+          </div>
+          <div class="project-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- About me section -->
+    <section class="about-wraper">
+      <div class="about">
+      <h1>About Me</h1>
+        <p>
+          I can help you build a product , feature or website, look through some of my work and experience!<br />
+          If you like what yousee and have a project you need coded, don’t hestiate to contact me.
+       </p>
+      </div>
+      <div class="about-button">
+        <button type="button">Get My Resume</button>
+      </div>
+      <!-- Languages -->
+      <div class="card simple-card">
+        <div class="image">
+          <img src="./img/icon-languages.png" alt="Languages">
+        </div>
+        <div class="outline-wrapper">
+          <div class="details">
+            <h2>Languages</h2>
+          </div>
+          <div class="items">
+            <div>Ruby on rails</div>
+            <div>CSS</div>
+            <div>JavaScript</div>
+            <div>html</div>
+          </div>
+        </div>
+      </div>
+      <!-- Frameworks -->
+      <div class="card simple-card">
+        <div class="image">
+          <img src="./img/icon-frameworks.png" alt="Framworks">
+        </div>
+        <div class="outline-wrapper">
+          <div class="details">
+            <h2>Framworks</h2>
+          </div>
+          <div class="items">
```

Comment:
- [ ] **Please use `ul` tags for grouping all the skills and `li` tags for each skill**

Again, for the same reason, as mentioned above. 😄